modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -643,7 +643,7 @@ def appointment_data(index = nil)
                                                           'status' => 'CANCELLED',
                                                           'status_detail' => 'CANCELLED BY CLINIC',
                                                           'time_zone' => 'America/Denver',
-                                                          'vetext_id' => nil,
+                                                          'vetext_id' => '442;3170516.',
```

Comment:
Have we confirmed that this is what we want? Having a dot with nothing after it seems strange?